packages/dds/tree/src/feature-libraries/modular-schema/modularChangeCodecs.ts

```diff
@@ -238,10 +239,24 @@ export function makeV0Codec(
 			  };
 	}
 
-	function decodeBuilds(
+	function encodeBuilds(
+		builds: ModularChangeset["builds"],
+		context: ChangeEncodingContext,
+	): EncodedBuilds | undefined {
+		return builds !== undefined ? encodeDetachedNodes(builds, context) : undefined;
+	}
+
+	function encodeRefreshers(
+		refreshers: ModularChangeset["refreshers"],
+		context: ChangeEncodingContext,
+	): EncodedBuilds | undefined {
+		return refreshers !== undefined ? encodeDetachedNodes(refreshers, context) : undefined;
+	}
```

Comment:
Why not have a single function?
Actually, it looks like `encodeDetachedNodes` already handles the case where `refreshers` is undefined, so it looks like these function are not needed at all (i.e., you can call `encodeDetachedNodes` directly). Am I missing something?

Reply:
I was wondering if we wanted these explicitly distinguished but if not, that sounds good to me!